package.json

```diff
@@ -61,10 +61,10 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "9.5.1",
-    "@grafana/experimental": "^1.1.0",
-    "@grafana/runtime": "9.5.1",
-    "@grafana/ui": "9.5.1",
+    "@grafana/data": "10.2.2",
```

Comment:
Can we do the package ugprade in a separate PR?

Reply:
Yes, I reverted this :)